src/Symfony/Component/Routing/Generator/Dumper/PhpGeneratorDumper.php

```diff
@@ -93,13 +93,18 @@ private function generateDeclaredRoutes()
             $properties[] = $compiledRoute->getHostTokens();
             $properties[] = $route->getSchemes();
 
-            $routes .= sprintf("        '%s' => %s,\n", $name, str_replace("\n", '', var_export($properties, true)));
+            $routes .= sprintf("                '%s' => %s,\n", $name, $this->exportArray($properties));
         }
-        $routes .= '    )';
+        $routes .= '            )';
 
         return $routes;
     }
 
+    protected function exportArray($array)
+    {
+        return preg_replace(array('/array \(/', '/,\n *\)/', '/,\n */', '/\n */'), array('array(', ')', ', ', ''), var_export($array, true));
```

Comment:
there is a risk of collision with the content of a string here, CS is not worth it imho